XbSymbolDatabase.c

```diff
@@ -1037,7 +1067,27 @@ bool XbSymbolScan(const void* xbeData, xb_symbol_register_t register_func)
 
                 if (bXRefFirstPass) {
                     if ((LibraryFlag & (XbSymbolLib_D3D8 | XbSymbolLib_D3D8LTCG)) > 0) {
-                        XbSymbolDX8SectionScan(LibraryFlag, pXbeHeader, BuildVersion, LibraryStr, register_func);
+                        for (unsigned int s = 0; s < pXbeHeader->dwSections; s++) {
+                            SectionName = (const char*)(xb_start_addr + pSectionHeaders[s].SectionNameAddr);
+                            if (strncmp(SectionName, Sec_D3D, 8) == 0) {
+
+                                if (isRunTime) {
+                                    // if an emulator did not load a section, then skip the section scan.
```

Comment:
"emulator"? Why is that relevant?

Reply:
It's relevant, except it is an xbe executable's responsibility to load a section. 😛